learning_observer/learning_observer/learning_tools/schoology.py

```diff
@@ -0,0 +1,152 @@
+import json
```

Comment:
I'm not going to specifically comment on things I already mentioned in the `learning_tools/canvas.py` file. Docstrings, pmss, etc. Apply that here and everywhere else.